index.js

```diff
@@ -197,20 +198,28 @@ async function underPressure (fastify, opts) {
   }
 
   async function onStatus (req, reply) {
+    const response = { status: 'ok' }
     if (healthCheck) {
       try {
-        if (!await healthCheck()) {
+        const checkResult = await healthCheck()
+        if (!checkResult) {
           req.log.error('external health check failed')
           reply.status(SERVICE_UNAVAILABLE).header('Retry-After', retryAfter)
           throw underPressureError
         }
+
+        return {
+          ...response,
+          ...checkResult
```

Comment:
minor: for consistency, I would either use spreading or Object.assign, since you've used Object.assign before, why use two different ways to do the same thing?